string name         #Name of path
nav_msgs/Path path  #List of path waypoints
